src/Componants/navbar.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import BottomNavigation from '@mui/material/BottomNavigation';
+import BottomNavigationAction from '@mui/material/BottomNavigationAction';
+import Homeicon from '@mui/icons-material/Home';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import AccountBoxIcon from '@mui/icons-material/AccountBox';
+
+export default function SimpleBottomNavigation() {
+  const [value, setValue] = React.useState(0);
+
+  return (
+    <Box sx={{ width: 500, position: 'fixed', bottom: 0, left: 0, right: 0 }}>
```

Comment:
Let's drop the explicit width so that it's responsive.
```suggestion
    <Box sx={{ position: 'fixed', bottom: 0, left: 0, right: 0 }}>
```